src/main/java/org/calrissian/flowmix/api/aggregator/AbstractGrouplessAggregator.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2015 Calrissian.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.calrissian.flowmix.api.aggregator;
+
+import static java.lang.System.currentTimeMillis;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import static java.util.UUID.randomUUID;
+import org.apache.commons.lang.StringUtils;
+import org.calrissian.flowmix.api.Aggregator;
+import org.calrissian.flowmix.core.model.event.AggregatedEvent;
+import org.calrissian.flowmix.core.support.window.WindowItem;
+import org.calrissian.mango.domain.Tuple;
+import org.calrissian.mango.domain.event.BaseEvent;
+import org.calrissian.mango.domain.event.Event;
+
+/**
+ *
+ * This abstract Aggregator doesn't mantain the whole group, you need to add and
+ * evict the tuples with simple operations and emit them with complex ones.
+ * 
+ * Currently under discussion
+ *
+ * @author Miguel A. Fuentes Buchholtz
+ * @param <T> Aggregation result type
+ * @param <F> Field type
+ */
+@Deprecated
```

Comment:
Not sure why we'd deprecate something that never existed in the codebase to begin with. Deprecation is to alert users that something is going away so they have time to change- things that have never been released should just be removed.